src/core/shelf.sol

```diff
@@ -124,7 +130,7 @@ contract Shelf is DSNote, TitleOwned {
     }
 
     function borrow(uint loan, uint wad) public owner(loan) {
-        require(nftLocked(loan), "nft-not-locked");
+        require(_nftLocked(loan), "nft-not-locked");
```

Comment:
I think nftLocked should be a public method. It can be helpful for other contracts and there is no harm in view only public methods.